test.py

```diff
@@ -167,10 +168,14 @@ def check_or_fail(condition,
 
 
 def check_version_found(plugin_version):
-    # This function should check that version in <p> element with class
-    # 'version' is equal to plugin version you provided with variable.
-    # You may check out one of other functions as example.
-    return False
+    try:
+        if plugin_version in driver.find_element_by_css_selector(
```

Comment:
hey, just woendering. 
Can you set `plugin_version` to `v0.1` will it pass or not?

Reply:
The test was throwing false positives indeed. Nice catch!

changed ```in``` for ```==```  and now the test fails with ```v0.1``` !